src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+
+import ProteinsListItem from './proteins-list-item/ProteinsListItem';
+
+import { fetchGene } from '../../../shared/rest/rest-data-fetchers/geneData';
+
+import { Gene } from '../../../types/gene';
+
+import styles from './ProteinsList.scss';
```

Comment:
Neither this file, nor `ProteinsListItemInfo.scss` are committed in this PR.

Reply:
✔️